### PR TITLE
address PR feedback

### PR DESCRIPTION
**Pull request by tbg**

### CODE REVIEW

File: client/sender.go [hunk 1]
```diff
@@ -93,7 +93,7 @@ func newSender(u *url.URL, ctx *base.Context, retryOptions retry.Options, stoppe
 // in that case.
 func SendWrapped(sender Sender, ctx context.Context, args roachpb.Request) (roachpb.Response, error) {
 	if ctx == nil {
```
Comment (tamird) [inline client/sender.go]: did we not want to remove this?

Comment (bdarnell) [inline client/sender.go]: Maybe, but removing it implies changing hundreds of lines that call this function with a `nil` context. For a convenience method I don't think it's worth it.

**Comment by bdarnell** [comment]
LGTM

File: client/sender.go (hunk 1 above)
Comment (tamird) [inline client/sender.go]: Sure.

**Pull request merged**